src/Table/index.tsx

```diff
@@ -25,5 +32,15 @@ export const StyledTable = styled(AntdTable)`
 export function Table<
   RecordType extends Record<string, unknown> = Record<string, unknown>,
 >(props: TableProps<RecordType>) {
-  return <StyledTable rowKey="id" {...props} />;
+  const { size } = useTheme();
+
+  return <StyledTable rowKey="id" size={mapSize(size)} {...props} />;
+}
+
+function mapSize(size: SizeType): SizeType {
+  // large is not designed by antd right now (though SizeType contains 'large')
+  if (size === 'large') {
+    return 'middle';
```

Comment:
```suggestion
    return 'default';
```
Anscheinend ist es der größte Wert vom `size`, siehe https://ant.design/components/table/#API.

Reply:
Found out that `large` is actually accepted, because the Table prop `size` is of type `SizeType` by AntD:

`export type SizeType = 'small' | 'middle' | 'large' | undefined;`

https://github.com/ant-design/ant-design/blob/master/components/config-provider/SizeContext.tsx

Which means the API documentation is a bit off and does not match the types.

Passing `default` leads to the same result as passing `large` - which means that values other than `small` and `middle` are just ignored by the table right now.